.forceignore

```diff
@@ -10,8 +10,9 @@ package.xml
 **profiles
 **settings
 
-
-**/__tests__/**
-**/test/jest-mocks/**
```

Comment:
Are things in the `force-app/test` directory automatically ignored?

Reply:
Yeah, because they are not in a metadata folder.